backend/config.py

```diff
@@ -63,10 +64,6 @@ def dataDirectory(self):
     def recipesDirectory(self):
         return '{0}/recipes/'.format(self.dataDirectory)
 
-    @property
-    def controllerHardwareDirectory(self):
```

Comment:
This is defined again below just above the `labHardwareDirectory` property decleration.